src/routes/formsgSiteCreation.ts

```diff
@@ -75,16 +77,18 @@ export class FormsgRouter {
         await this.sendCreateError(requesterEmail, repoName, submissionId, err)
         return res.sendStatus(200)
       }
-      const foundUser = await this.usersService.findByEmail(requesterEmail)
-      if (!foundUser) {
+      const foundRequester = await this.usersService.findByEmail(requesterEmail)
+      if (!foundRequester) {
         const err = `Form submitter ${requesterEmail} is not an Isomer user. Register an account for this user and try again.`
         await this.sendCreateError(requesterEmail, repoName, submissionId, err)
```

Comment:
sorry a bit confused, this is an error flow right? Why are we returning 200 res after sendCreateError?

Reply:
This endpoint is only triggered by forms, which expects a 200 response! The error handling is done separately via email